src/patternfly/demos/Toolbar/examples/Toolbar.md

```diff
@@ -52,7 +52,7 @@ import './Toolbar.css'
       {{#> toolbar-item toolbar-item--modifier=(concat 'pf-m-pagination ' toolbar-item-pagination--modifier)}}
         {{#> pagination pagination--modifier="pf-m-compact pf-m-hidden pf-m-visible-on-md"}}
           {{#> pagination pagination--IsCompact="true"}}
-            {{> pagination-options-menu id="pagination-options-menu-bottom-example"}}
```

Comment:
Same comment as mentioned about the select component for updating `id` to `options-menu--id`. Sharing the same partial parameter name can cause issues.

Reply:
Created https://github.com/patternfly/patternfly/issues/5000 as follow-up to track this request.